backend/routes/events.js

```diff
@@ -71,7 +71,35 @@ router.get('/admin/all', async (req, res, next) => {
     } catch (err) {
         handleError(err, req, res, next);
     }
+});
+
+//Get all events  (non-admin)
+router.get('/non_admin/dashboard', async (req, res, next) => {
+    try {
+        if (!req.user) {
+            throw new Error('401__You must be logged in');
+        } else {
+            const userId = processInput(req.user.a_id || req.user.v_id || req.user.f_id, "idNum", "user id");
+            let usertype = null;
+            if (req.user.a_id) {
+                usertype = "staff";
+            } else if (req.user.v_id) {
+                usertype = "volunteer";
+            } else if (req.user.f_id) {
+                usertype = "fellow";
+            }
 
+            const dashboardEvents = await eventsQueries.getAllEventsNonadmin(usertype, userId);
```

Comment:
and in this case it will be `const dashboardEvents = await eventsQueries.getAllEventsVolunteer(req.user.v_id)`